src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -123,15 +125,30 @@ static void Main(string[] args)
             if (!Boolean.TryParse(teamCityFormattingOn, out TeamCityFormattingOn))
                 throw new ArgumentException("Invalid argument provided for -Dteamcity");
 
-            SinkCoordinator = TestRunSystem.ActorOf(TeamCityFormattingOn ?
-                Props.Create(() => new SinkCoordinator(new[] { new TeamCityMessageSink(Console.WriteLine, suiteName) })) : // mutes ConsoleMessageSinkActor
-                Props.Create<SinkCoordinator>(), "sinkCoordinator");
 
             var listenAddress = IPAddress.Parse(CommandLine.GetPropertyOrDefault("multinode.listen-address", "127.0.0.1"));
             var listenPort = CommandLine.GetInt32OrDefault("multinode.listen-port", 6577);
             var listenEndpoint = new IPEndPoint(listenAddress, listenPort);
             var specName = CommandLine.GetPropertyOrDefault("multinode.spec", "");
             var platform = CommandLine.GetPropertyOrDefault("multinode.platform", "net");
+            var reporter = CommandLine.GetPropertyOrDefault("multinode.reporter", "trx");
+
+            Props coordinatorProps;
+            switch (reporter.ToLowerInvariant())
```

Comment:
Need to have an option for the console formatter, which is what would occur in the case that this value was never provided.